graphql/tests/mutation/saveAdminSettings/queries/save.graphql

```diff
@@ -2,6 +2,11 @@ mutation{
     saveAdminSettings(adminSettings:  {
          banner: "New Banner"
          bannerTheme: ANNOUNCEMENT
+         serviceFlags: {
```

Comment:
nit. probably have to fix this test, I'm not seeing the values get updated in practice 🤔